src/store/ducks/tests/goals.test.ts

```diff
@@ -37,7 +37,7 @@ describe('reducers/goals', () => {
   });
 
   it('should fetch goals', () => {
-    store.dispatch(fetchGoals(fixtures.goals));
+    store.dispatch(fetchGoals(fixtures.goals as Goal[]));
```

Comment:
Should these types be defined when `fixtures.goals` is defined?

Reply:
For some reason Typescript cannot infer what type fixtures.goals is. I decided to do this after trying and failing to change the fixture.